### PR TITLE
py2cairo 1.10.0: Fix for linuxbrew

### PR DESCRIPTION
**Pull request by paulhybryant**
Fix the error: No module _cairo can be found
The error is only triggered when the python module is used.

### CODE REVIEW

**Comment by tdsmith** [comment]
Using the pyext mode is probably correct on Linux and should let you skip the renaming dance, if you'd rather go that route.

**Comment by paulhybryant** [comment]
Can you elaborate more about pyext

File: Library/Formula/py2cairo.rb [hunk 1]
```diff
@@ -30,7 +30,7 @@ def install
     # disable waf's python extension mode because it explicitly links libpython
     # https://code.google.com/p/waf/issues/detail?id=1531
     inreplace "src/wscript", "pyext", ""
-    ENV["LINKFLAGS"] = "-undefined dynamic_lookup"
+    ENV["LINKFLAGS"] = "-undefined dynamic_lookup" if OS.mac?
     ENV.append_to_cflags `python-config --includes`
```
Comment (tdsmith) [inline Library/Formula/py2cairo.rb]: You can probably remove these there lines (or make them conditional on `OS.mac?`) and then get rid of the `mv` below.

**Comment by paulhybryant** [comment]
I have updated the change according to tdsmith's comment.
It works fine in my test.
Thanks,

**Comment by sjackman** [comment]
Instead of deleting this code, please move it inside `if OS.mac?`.
Please see https://github.com/Homebrew/linuxbrew/blob/master/CONTRIBUTING.md

**Comment by paulhybryant** [comment]
Sure. Updated.

**Comment by sjackman** [comment]
Merged. Thanks for your contribution, Paul.

**Pull request closed without merging**